app/views/people/_contact_details.html.haml

```diff
@@ -7,11 +7,11 @@
         - if contact_point.email.present?
           %dt= t('contact_points.email').capitalize + ": "
           %dd
-            %a{ href: "mailto:#{contact_point.email}" }= contact_point.email
+            = link_to(contact_point.email, "mailto:#{contact_point.email}")
```

Comment:
Also possible to use the [mail_to](http://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/UrlHelper.html#method-i-mail_to) method